src/lib.rs

```diff
@@ -188,6 +189,75 @@ impl FixedBitSet
     {
         &mut self.data
     }
+
+    /// Iterates over all set bits, i.e. all '1's.
+    #[inline]
+    pub fn iter_ones<'a>(&'a self) -> IterOnes<'a> {
```

Comment:
The method doc should explain explicitly what the iterator element is / what it means.